### PR TITLE
Deployment stacks RELNOTES

### PR DESCRIPTION
**Pull request by timwarner-msft**
#assign-reviewer:@apclouds

Angel, this is a skeleton release notes doc. If you want to send me a list of bullet points, I'm happy to integrate them into the document.

cc: @bmoore-msft

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+> Includes updates/fixes to the client-side modules for Deployment Stacks Private Preview.
+> Individuals/organizations with Azure subscriptions onboarded for the Private Preview can
+> download/extract this release and follow the installation instructions within the [README](./README.md).
+
+### Upgrade steps
+
+- <TODO>
+
+### Breaking changes
+
```
Comment (aperezmicrosoft) [inline RELEASE_NOTES.md:10]: Removed Snapshots and changed command names

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- <TODO>-
+
+### New features
+
```
Comment (aperezmicrosoft) [inline RELEASE_NOTES.md:15]: Bulk delete, and ExportStack Template (before it was GetStack api), DenyAssignments* (DenyWriteDelete still in progress), MG Support, new UpdateBehavior and ActionOnUnmanage

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- Changed Azure PowerShell and Azure CLI command names
+
+### New features
+
+- Bulk delete
+- Export stack template (before you needed to use the GetStack API directly)
+- Deny assignments (`DenyWriteDelete` is still in progress)
+- Management group support
+- New `-UpdateBehavior` and `-ActionOnUnmanage` parameter values
+
```
Comment (alex-frankel) [inline RELEASE_NOTES.md:22]: Can we also call out the overall UX improvements to PS/CLI as a new feature? IMO, that is one of the biggest features of this release. I see at called out on Line 5, but I don't think that is where I would expect to see it/doesn't put enough emphasis on it.

Comment (timwarner-msft) [reply RELEASE_NOTES.md:22]: done

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Deployment stacks release notes
+
+## [Deployment Stacks Private Preview - Client Packages v0.1.7 (January 11, 2023)
```
Comment (stuartko) [inline RELEASE_NOTES.md]: Is the bracket here ("[") unintentional, or was it meant to be a link to another MD?

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+- Removed snapshots
+- Changed Azure PowerShell and Azure CLI command names
+
+### New features
+
+- Significant improvements to the deployment stacks Azure PowerShell and Azure CLI commands
+- Bulk delete
+- Export stack template (before you needed to use the GetStack API directly)
+- Deny assignments (`DenyWriteDelete` is still in progress)
+- Management group support
```
Comment (stuartko) [inline RELEASE_NOTES.md]: Suggest we extend this a bit to be more specific. Something like "Stacks can now be scoped at Management Groups", or "Management group scoping support"?

**Pull request closed without merging**